lib/cursor-position-view.coffee

```diff
@@ -22,7 +26,9 @@ class CursorPositionView extends HTMLElement
 
   updatePosition: ->
     if position = @getActiveTextEditor()?.getCursorBufferPosition()
-      @textContent = "#{position.row + 1}:#{position.column + 1}"
+      @row = position.row + 1
+      @column = position.column + 1
+      @textContent = "#{@row}:#{@column}"
     else
       @textContent = ''
```

Comment:
Yup, it's defined here: https://github.com/atom/status-bar/blob/master/styles/status-bar.less#L80-L85